src/use-hotkeys/use-hotkeys.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller } from 'stimulus'
+// @ts-ignore
+import hotkeys from 'hotkeys-js'
+
+export const bindHotkeys = (controller: Controller) => {
```

Comment:
I think we should keep the `use` naming scheme throughout the library. Probably `useHotkeys` would fit better?